daisy_workflows/image_build/enterprise_linux/build_installer.py

```diff
@@ -32,6 +32,9 @@ def main():
   # Get Parameters
   release = utils.GetMetadataAttribute('el_release', raise_on_not_found=True)
   savelogs = utils.GetMetadataAttribute('el_savelogs') == 'true'
+  install_disk = 'scsi-0Google_PersistentDisk_'
```

Comment:
In this environment, we have symlinks to the device name which come from this [udev rule](https://github.com/GoogleCloudPlatform/guest-configs/blob/master/src/lib/udev/rules.d/65-gce-disk-naming.rules). It is probably better to use these symlinks which will exist in this VM environment (it's not quite true in the anaconda build env though) to deal with the NVMe machine types.

Reply:
The disk reference isn't used in this environment, it's used in the anaconda environment. I will copy the udev rule into there and expand this to handle nvme machine types too.

Reply:
If you want to see an NVMe case- you can use the arm build which is NVMe only.

Reply:
Done.